ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host2/ip-dhcp-snooping.yml

```diff
@@ -0,0 +1,5 @@
+---
+### IP DHCP Snooping ###
+
+ip_dhcp_snooping:
+  enabled: true
```

Comment:
should not this one be false for coverage?

Reply:
I see - yeah these are weird branching to add but understood - thanks for taking the time!